.github/workflows/ci-actions-incremental.yml

```diff
@@ -54,6 +54,10 @@ env:
   COMMON_TEST_MAVEN_ARGS: "-Dformat.skip -Denforcer.skip -DskipDocs -Dforbiddenapis.skip -DskipExtensionValidation -DskipCodestartValidation"
   NATIVE_TEST_MAVEN_ARGS: "-Dtest-containers -Dstart-containers -Dquarkus.native.native-image-xmx=6g -Dnative -Dnative.surefire.skip -Dno-descriptor-tests clean install -DskipDocs"
   JVM_TEST_MAVEN_ARGS: "-Dtest-containers -Dstart-containers -Dquarkus.test.hang-detection-timeout=300"
+  # Important: keep these selectors in sync with the grep commands in the calc_run_flags job!
+  # This may be a lot better with maven 4, but with maven 3, excluding a project does not exclude its children, and it's not possible to include a project and explicitly exclude some children; compensate by doing excludes the low-tech way, at the shell level
+  JVM_TEST_INTEGRATION_TESTS_SELECTOR: "-f integration-tests -pl !gradle -pl !maven -pl !devmode -pl !devtools"
```

Comment:
Does this work `-pl !gradle -pl !maven -pl !devmode -pl !devtools`? I would expect the whole `artifactId` to be needed?

Or when you do a `-f`, you can reference them with the nested directory name?

Reply:
> Or when you do a -f, you can reference them with the nested directory name?

Exactly that. Because of the -f, the names are resolved relative to the 'new' root module. I originally tried `-f !integration-tests/gradle` and got all sorts of errors about modules not being present in the reactor, which then led me down the route of using scripts and stuff. I think artifact id would also work, but the path is what we usually use, because it's shorter.